### PR TITLE
Add a.DI CrossChainController event

### PR DESCRIPTION
**Pull request by katamarinaki**
This PR contains an addition of new alert for CrossChainController, which is a part of a.DI contracts set

Links: [LIP-24](https://github.com/lidofinance/lido-improvement-proposals/blob/develop/LIPS/lip-24.md), [a.DI repo](https://github.com/lidofinance/aave-delivery-infrastructure)

### CODE REVIEW

File: l2-bridge-ethereum/src/constants.ts [hunk 1]
```diff
@@ -107,6 +107,8 @@ export const ARBITRUM_GATEWAY_SET_EVENT =
 export const LINEA_CUSTOM_CONTRACT_SET_EVENT =
   "event CustomContractSet(address indexed nativeToken, address indexed customContract, address indexed setBy);";
 
+export const BSC_CROSS_CHAIN_CONTROLLER = "";
```
Comment (arwer13) [inline l2-bridge-ethereum/src/constants.ts:110]: Please add L1 to the name for clarity
```suggestion
export const BSC_L1_CROSS_CHAIN_CONTROLLER = "";
```

**Review by arwer13 — APPROVED** [review]
lgtm

**Comment by arwer13** [comment]
Have got a question, are alerts for e.g. `BridgeAdapterUpdated`, `OwnershipTransferred` events needed as well? Could you please provide a list of all related alerts and maybe group them into categories like "needed" and "not needed because ..."?

File: l2-bridge-ethereum/src/constants.ts (hunk 1 above)
Comment (katamarinaki) [reply l2-bridge-ethereum/src/constants.ts]: Good catch, thanks. Addressed in [63574c0](https://github.com/lidofinance/alerting-forta/pull/564/commits/63574c096df8db69e1203cfe6b200e01d6169f31).

**Comment by katamarinaki** [comment]
@arwer13 hey, it is the only alert needed for a deploy. There are two more alerts that we need to watch for on Ethereum, but I'll get them done later.

**Review by tamtamchik — APPROVED** [review]

**Pull request merged**